.github/workflows/PR.yml

```diff
@@ -105,4 +105,4 @@ jobs:
           report_paths: '**/build/test-results/test/TEST-*.xml'
 
 env:
-  GRADLE_OPTS: -Dorg.gradle.parallel=true
+  GRADLE_OPTS: -Dorg.gradle.parallel=true -Dorg.gradle.caching=true
```

Comment:
Do I really need to enable the caching explicitly? 🤔

Reply:
According to this [thread](https://gradle-community.slack.com/archives/CAHSN3LDN/p1673970310498069), we do have to enable the cache explicitly to get the task outputs.